apps/web/components/sections/faqs/faq-header.tsx

```
@@ -0,0 +1,59 @@
+						value={inputValue}
+						onChange={handleSearch}
+						className="pl-8 text-xs md:text-base md:pl-10"
+					/>
+					{filteredQuestions.length > 0 && (
+						<div className="absolute top-full left-0 w-full bg-white border border-gray-300 shadow-lg rounded-lg z-50 max-h-60 overflow-y-auto">
+							{filteredQuestions.map((faq) => (
+								<p
+									key={faq.id}
+									className="text-black text-xs p-3 border-b last:border-none cursor-pointer hover:bg-gray-100 md:text-base"
+									onClick={() => handleSelectQuestion(faq)}
+								>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)